src/launchpad/utils/android/bundletool.py

```diff
@@ -46,17 +47,10 @@ def __init__(self, bundletool_path: str | Path | None = None) -> None:
             FileNotFoundError: If bundletool cannot be found at specified path or in PATH
         """
         if bundletool_path is None:
-            try:
-                # TODO: Ensure packaged in docker image when productionizing
-                result = subprocess.run(
-                    ["which", "bundletool"],
-                    capture_output=True,
-                    text=True,
-                    check=True,
-                )
-                bundletool_path = result.stdout.strip()
-            except subprocess.CalledProcessError as e:
-                raise FileNotFoundError("bundletool not found in PATH. Install with `brew install bundletool`") from e
+            # TODO: Ensure packaged in docker image when productionizing
+            bundletool_path = shutil.which("bundletool")
+            if bundletool_path is None:
+                raise FileNotFoundError("bundletool not found in PATH. Install with `brew install bundletool`")
```

Comment:
I think this if condition can be removed.
- `brew install bundletool` is not the right way to fix this now (devs should run `make install-dev`)
- Missing bundletool is handled below with the `FileNotFoundError`